purchases/src/main/kotlin/com/revenuecat/purchases/Offerings.kt

```diff
@@ -9,6 +9,7 @@ package com.revenuecat.purchases
 data class Offerings(
     val current: Offering?,
     val all: Map<String, Offering>,
+    val placements: Placements?,
```

Comment:
Hmm interesting... since the public constructor here has `Placements` which is `private`, I would think it shouldn't compile here?

I think the way to do this is to make the constructor for `Offerings` `internal` and then add another constructor without a `placements` parameter to keep retrocompastibility.

Reply:
See my comment in: https://github.com/RevenueCat/purchases-android/pull/1604#discussion_r1496127913